rsmtool/utils.py

```diff
@@ -648,20 +648,19 @@ def get_output_directory_extension(directory, experiment_id):
         and are in the list of possible output extensions.
     """
     extension = 'csv'
-    extensions_identified = [ext for ext in POSSIBLE_EXTENSIONS
-                             if has_files_with_extension(directory, ext)]
+    extensions_identified = set(ext for ext in POSSIBLE_EXTENSIONS
```

Comment:
Sorry, you can just use `{ .. }` instead of `[ .. ]` and that just gives you a set comprehension. I thought I had mentioned that in my earlier comment.